### PR TITLE
introduced group admin view

### LINKED ISSUE #647: Failing group membership changes
Changing which users belong to a group in the respective admin menu fails with a `400` server return message.

### PR DESCRIPTION
**Pull request by jeriox**
closes #647

### CODE REVIEW

File: _1327/user_management/forms.py [hunk 1]
```diff
@@ -47,3 +49,25 @@ class UserImpersonationForm(forms.Form):
...
+	add_minutes = BooleanField(required=False)
+	add_poll = BooleanField(required=False)
+	users = ModelMultipleChoiceField(UserProfile.objects.all())
+
+	class Meta:
+		model = Group
+		fields = ("name",)
+
+	def __init__(self, *args, **kwargs):
+		super().__init__(**kwargs)
```
Comment (janno42) [inline _1327/user_management/forms.py]: you need to replace this with ```super(GroupEdit, self).__init__(*args, **kwargs)```, then the form should be valid on submission.

File: _1327/user_management/forms.py [hunk 2]
```diff
@@ -47,3 +49,25 @@ class UserImpersonationForm(forms.Form):
 	def __init__(self, *args, **kwargs):
 		super(UserImpersonationForm, self).__init__(*args, **kwargs)
 		self.fields['username'].widget.attrs['class'] = 'select2-selection'
+
+
+class GroupEdit(forms.ModelForm):
```
Comment (janno42) [inline _1327/user_management/forms.py]: please rename the form to `GroupEditForm` to make clear that it is a form :)

**Comment by jeriox** [comment]
@janno42 Apparently it is not possible to provide inital values to custom Fields on a ModelForm from the model. Thus I re-initialize the fields in the init method. Can you think of a better way for that?

**Comment by janno42** [comment]
setting the values in the init method is perfectly fine.

**Pull request merged**